src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -634,9 +649,19 @@ export default class GeFormRenderer extends LightningElement{
             const formControls = this.getFormControls(event);
             formControls.toggleSpinner();
 
+            let tokenizedGift = null;
             try {
                 if (this.shouldTokenizeCard()) {
-                    await this.tokenizeCard(sectionsList)
+                    tokenizedGift = new ElevateTokenizeableGift(
+                        this.cardholderNames.firstName,
+                        this.cardholderNames.lastName,
```

Comment:
Could we just pass in `this.cardholderNames` and parse out what we need in the constructor?